Makefile

```diff
@@ -213,11 +213,15 @@ aws-build-and-deploy-aws-data-clear-lambda: build-backend aws-upload-artifacts a
 # AWS Interactions
 # ======================================================================
 
+	
+
 aws-sync-files-from-prod-to-dev:
-	@aws s3 sync s3://pcc-integration-data-files-prod s3://pcc-integration-data-files-dev --acl bucket-owner-full-control
+	@aws s3 sync s3://pcc-integration-data-files-prod/sbc s3://pcc-integration-data-files-dev/sbc --acl bucket-owner-full-control
```

Comment:
This change is not related to the bug, but it prevents the archived directory from syncing (this causes an error to be printed due to an issue with a file in our archive)

Reply:
Seems reasonable not to sync the archive - we might want to mention this to Qudsia too just in case, but if we need to sync that we can make a new ticket for it 😄